Framework/Algorithms/src/CreateDetectorTable.cpp

```diff
@@ -135,8 +135,11 @@ ITableWorkspace_sptr createDetectorTableWorkspace(const MatrixWorkspace_sptr &ws
     hasDiffConstants = true;
   }
 
+  auto wsName = ws->getName();
```

Comment:
I'm not sure this workspace name check is necessary? If we look at the table population code:

https://github.com/mantidproject/mantid/blob/44f0585b10f58a9e6e458012a4e21459fa2b19b3/Framework/Algorithms/src/CreateDetectorTable.cpp#L281


then, as Spencer says, the value is calculated to be elastic all of the time so can we just update the column title to say Q Elastic and not check the name?

Reply:
Having spoken with Spencer we worked out that we can have it listed as Q Elastic for all cases.

Reply:
Thanks.